index.d.ts

```diff
@@ -22,6 +22,11 @@ declare const SecureSessionPlugin: FastifyPlugin<SecureSessionPluginOptions>;
 export default SecureSessionPlugin;
 
 declare module "fastify" {
+  interface FastifyInstance {
+    decodeSecureSession(request: FastifyRequest): Session | null
+    encodeSecureSession(session: Session): string
+  }
```

Comment:
Can you please add a test for the types?

Reply:
Added